gem.json

```diff
@@ -1,6 +1,6 @@
 {
     "gem_name": "HumanWorker",
-    "version": "2.0.0",
+    "version": "2.1.0",
```

Comment:
No need to bump the minor, we bumped the major already between the last release and the next release.